### PR TITLE
fix: stderr forward if no stderr exists

### LINKED ISSUE #471: Interactive mode (REPL)  (v 1.8.0a1) Error
**from playwright.sync_api import sync_playwright
playwright = sync_playwright().start()**

Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\playwright\_impl\_transport.py", line 27, in _get_stderr_fileno
    return sys.stderr.fileno()
io.UnsupportedOperation: fileno

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<pyshell#1>", line 1, in <module>
    playwright = sync_playwright().start()
  File "C:\Python37\lib\site-packages\playwright\sync_api\_context_manager.py", line 73, in start
    return self.__enter__()
  File "C:\Python37\lib\site-packages\playwright\sync_api\_context_manager.py", line 67, in __enter__
    dispatcher_fiber.switch()
  File "C:\Python37\lib\site-packages\playwright\sync_api\_context_manager.py", line 48, in greenlet_main
    loop.run_until_complete(self._connection.run_as_sync())
  File "C:\Python37\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "C:\Python37\lib\site-packages\playwright\_impl\_connection.py", line 134, in run_as_sync
    await self.run()
  File "C:\Python37\lib\site-packages\playwright\_impl\_connection.py", line 139, in run
    await self._transport.run()
  File "C:\Python37\lib\site-packages\playwright\_impl\_transport.py", line 62, in run
    stderr=_get_stderr_fileno(),
  File "C:\Python37\lib\site-packages\playwright\_impl\_transport.py", line 34, in _get_stderr_fileno
    return sys.__stderr__.fileno()
AttributeError: 'NoneType' object has no attribute 'fileno'

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes #471

### CODE REVIEW

File: playwright/_impl/_transport.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def _get_stderr_fileno() -> Optional[int]:
         # pytest-xdist monkeypatches sys.stderr with an object that is not an actual file.
         # https://docs.python.org/3/library/faulthandler.html#issue-with-file-descriptors
         # This is potentially dangerous, but the best we can do.
-        if not hasattr(sys, "__stderr__"):
+        if not hasattr(sys, "__stderr__") or not sys.__stderr:  # type: ignore
```
Comment (kumaraditya303) [inline playwright/_impl/_transport.py:32]:
```suggestion
        if not hasattr(sys, "__stderr__") or not sys.__stderr__: 
```

Comment (mxschmitt) [reply playwright/_impl/_transport.py:32]: See here, there is the `__stderr__` attribute but it is `None`: https://github.com/microsoft/playwright-python/issues/471#issue-797853555. I try to handle this case with this PR.

Comment (kumaraditya303) [reply playwright/_impl/_transport.py:32]: Your PR is checking `for __stderr` instead of `__stderr__`

Comment (mxschmitt) [reply playwright/_impl/_transport.py:32]: good catch!

**Comment by pavelfeldman** [comment]
Looks good, but we need to verify it!

**Review by pavelfeldman — APPROVED** [review]

File: playwright/_impl/_transport.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def _get_stderr_fileno() -> Optional[int]:
         # pytest-xdist monkeypatches sys.stderr with an object that is not an actual file.
         # https://docs.python.org/3/library/faulthandler.html#issue-with-file-descriptors
         # This is potentially dangerous, but the best we can do.
-        if not hasattr(sys, "__stderr__"):
+        if not hasattr(sys, "__stderr__") or not sys.__stderr__:
             return None
         return sys.__stderr__.fileno()
```
Comment (pavelfeldman) [inline playwright/_impl/_transport.py:34]: I bet you have sys.__stderr__, but don't have sys.__stderr__.fileno here.

**Pull request merged**